PR Week 2 Alejandro Rivas

aerr96 opened the pull request:
# Class 3 Week 2 exercise:
This first commit contains semantic updates to the week1 index HTML file.
1. Last repository files were copied and renamed according to the specifications in the "semana-2" cloned repo.
2. Created the global article "Profile" containing the main header and a new section called "Skills and Experience." adding as an h2 title.
3. New Section Skills contains the three subject previews created with the possibilities to be expanded as an article each.
4. Modified the previews h2 titles to h3 type.
5. Stablished a new section inside the `` <aside> `` block named Aside Content.
6. In The "aside" section, the CLI table was modified to have borders and text alignment in the styles CSS file.
7. Added footer section of the whole page, containing the contact info and email address.

pacorroX reviewed (changes requested):
Parece que falta la sección/página de contacto de [los ejercicios de la Clase 4](https://github.com/undefined-academy/semana-2#clase-4-ejercicios).

aerr96 commented:
# Class 4 Week 2 exercise:
With the last commit a Contact Form page was added 
1. Related to the example showed in class, a contact form page was created in a different html file. Can be accessed in the footer section of the main page.
2. Added labels and inputs for: Name, Last Name, Email, Company, Phone Number, Budget, Project Type, Description, Appointment, and terms validation checkbox.
3. CCS file is shared with the main page, and with the div class `` form-fields `` all elements of the form share some visual characteristics. 
4. Also implemented color effects in the button for a better user experience.
5. Finally, an exception was added to prevent the default HTML form behavior of the server solicitude pressing the send button, with the JS sentence in the `` <button> `` tag: `` <button type="button" onclick="document.getElementById('my-form').onsubmit(function(e) {e.preventDefault();})">Send</button> ``

aerr96 commented:
> Parece que falta la sección/página de contacto de [los ejercicios de la Clase 4](https://github.com/undefined-academy/semana-2#clase-4-ejercicios). Entiendo que lo harás en otro PR.

Correcto, justo acabo de hacer el commit, y los comentarios de los cambios finales de este repositorio para la semana 2. Gracias por tu revisión.
